packages/xrpl/package.json

```diff
@@ -67,6 +65,13 @@
   },
   "readmeFilename": "README.md",
   "engines": {
-    "node": ">=10.13.0"
+    "node": ">=14"
+  },
+  "devDependencies": {
+    "@geut/browser-node-core": "^2.0.13",
+    "@types/node": "^18.11.9",
```

Comment:
I think this should be `^14.14.31` - which as far as I know is the most up to date version which matches Node 14 types since that's the oldest version of Node we support currently. (That way we get errors if we try to use types which are exclusive to higher versions of Node) 

(Unless it's fine for tests? I'm not sure how lerna works with dev dependencies in the sub-packages - I thought they were all supposed to be hoisted up to the top-level package if they were a dev dependency, but I could be wrong)

Reply:
Updated